modules/installation-osp-kuryr-config-yaml.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-kuryr-config-yaml_{context}"]
+= Sample customized `install-config.yaml` file for OpenStack with Kuryr
+
+To deploy with Kuryr SDN instead of the default OpenShift SDN, you must
+modify the `install-config.yaml` file to include `Kuryr` as the desired
+`networking.networkType` and proceed with the the default OpenShift SDN steps.
```

Comment:
Should the word "installation" be added to this line? "proceed with the the default OpenShift SDN **installation** steps."